app/src/main/java/co/touchlab/kampkit/android/metaweather/view/ViewSideEffects.kt

```diff
@@ -0,0 +1,7 @@
+package co.touchlab.kampkit.android.metaweather.view
+
+import co.touchlab.kampkit.android.metaweather.States
+
+data class ViewSideEffects(
+    val text: String = "",
+)
```

Comment:
I'm not sure how I feel about keeping around SideEffects for the sake of testing them out. Let's remove it, now that we've verified it's functionality.